lexicant.html

```diff
@@ -613,18 +613,21 @@ <h1>Lexicant game</h1>
 
     function SelectOpponent({ opponent, setOpponent }) {
       return (
-        <select
-          value={opponent}
-          onChange={(ev) => setOpponent(ev.target.value)}
-        >
-          <option value="default" disabled>
+        <div>
+          <span>
             Choose your opponent...
-          </option>
-          <option value="Random">Random Randi</option>
-          <option value="Careful">Careful Carter</option>
-          <option value="Perfect">Perfect Percy</option>
-          <option value="Trigram">Trigram Tracy</option>
-        </select>
+            <br />
```

Comment:
Typically we want the label of a component to be semantically marked in the HTML, I would suggest using a `<label>` here with its `htmlFor` attribute set to the `<select>` below. See for example: https://react.dev/reference/react-dom/components/select#providing-a-label-for-a-select-box